frontend/.env.development

```diff
@@ -1 +1,2 @@
-VITE_CONFIG_URL=config.development.json
\ No newline at end of file
+VITE_CONFIG_URL=config.development.json
+VITE_FAUCET_ENABLED=true
```

Comment:
I was thinking about it. The simple question is: based on what?

Reply:
Alright, so we discussed this topic in our Daily meeting.
Ivan thought it might be possible to identify which layer 1 the roll-ups are connected to and take this. But there is no general approach to get this information. There might be a way by using the Beamer contracts itself. Though this would be arguably more complete solution, it is asynchronous and quite laborious. Putting both solutions against each other, it is quite hard to reasoning the effort. At least in the current context of solving this problem before going to mainnet. This means the configuration approach is the choice for now, but in future we might come up with an automated solution.